arho_feature_template/gui/dialogs/plan_feature_form.py

```diff
@@ -114,7 +114,12 @@ def _check_regulation_group_short_names(self) -> bool:
             if not short_name:
                 continue
 
-            if short_name in self.existing_group_short_names:
+            if (
+                not (
+                    reg_group_widget.regulation_group.id_ and short_name == reg_group_widget.regulation_group.short_name
+                )
+                and short_name in self.existing_group_short_names
+            ):
```

Comment:
Mun oli vaikea ymmärtää mitä tässä tarkistetaan. Saisiko ehtoa selkeytettyä jotenkin näin?
```suggestion
            def is_existing_model_with_unmodified_short_name(model, short_name):
                return model.id_ and short_name == model.short_name
            if (
                not is_existing_model_with_unmodified_short_name(reg_group_widget.regulation_group, short_name)
                and short_name in self.existing_group_short_names
            ):
```